gsa/src/web/entity/__tests__/box.js

```diff
@@ -17,22 +17,29 @@
  */
 import React from 'react';
 
-import Date, {setLocale} from 'gmp/models/date';
+import date, {setLocale} from 'gmp/models/date';
```

Comment:
This was upper case on purpose, because it's a component model, just like we have Result, Target etc in upper case as well. Putting this into lower case might give the impression, this is some kind of transformation function.